lib/attr.gd

```diff
@@ -37,8 +37,14 @@
 ##
 ##  <Description>
 ##  This info class (together with <Ref InfoClass="InfoWarning"/> is used
```

Comment:
Optional suggestion:
```suggestion
##  This info class (together with <Ref InfoClass="InfoWarning"/>) is used
```
Missing closing parenthesis (not you fault, of course, but in case other changes need to be made to this PR, this could also be addressed.